src/pages/Homepage/Homepage.vue

```diff
@@ -33,12 +34,14 @@ export default {
 	components: {
 		WwwPage,
 		DefaultHomePage,
+		LendByCategoryHomepage,
 		IWDHomePage,
 		WRDHomePage,
 		TopMessageContentful,
 	},
 	data() {
 		return {
+			isLendByCategoryActive: false,
```

Comment:
@mlibazisi When you build the experiment, setting this to true will show the lend-by-category homepage.